src/test/java/com/hubspot/jinjava/lib/exptest/isDivisibleByExpTestTest.java

```diff
@@ -0,0 +1,78 @@
+package com.hubspot.jinjava.lib.exptest;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.Assert.assertEquals;
+
+import com.hubspot.jinjava.Jinjava;
+import java.util.HashMap;
+import org.junit.Before;
+import org.junit.Test;
+
+public class isDivisibleByExpTestTest {
+  private static final String DIVISIBLE_BY_TEMPLATE = "{{ %s is divisibleby %s }}";
+
+  private Jinjava jinjava;
+
+  @Before
+  public void setup() {
+    jinjava = new Jinjava();
+  }
+
+  @Test
+  public void itRequiresDividend() {
+    assertEquals(
+      jinjava.render(String.format(DIVISIBLE_BY_TEMPLATE, "null", "3"), new HashMap<>()),
+      "false"
+    );
+  }
+
+  @Test
+  public void itRequiresNumericalDividend() {
+    assertEquals(
+      jinjava.render(
+        String.format(DIVISIBLE_BY_TEMPLATE, "thirty", "3"),
+        new HashMap<>()
+      ),
+      "false"
+    );
+  }
+
+  @Test
+  public void itRequiresDivisor() {
+    assertEquals(
+      jinjava.render(String.format(DIVISIBLE_BY_TEMPLATE, "10", "null"), new HashMap<>()),
+      "false"
+    );
+  }
+
+  @Test
+  public void itRequiresNumericalDivisor() {
+    // Question for PR: It seems that InvalidArgumentExceptions are caught before my middleware. How to know it failed for the right reason?
```

Comment:
you can look at the errors on the interpreter by using `jinjava.renderForResult`. [Example](https://github.com/HubSpot/jinjava/blob/c2d7df73cc9b91c90ee0ea4c89f85dcb3abbe532/src/test/java/com/hubspot/jinjava/lib/filter/SortFilterTest.java#L92)

Reply:
Yes. I followed your suggestion and this addresses my need.